packages/fxa-profile-server/lib/routes/_core_profile.js

```diff
@@ -54,6 +55,7 @@ module.exports = {
               Authorization: 'Bearer ' + req.auth.credentials.token,
             },
             json: true,
+            timeout: AUTH_SERVER_TIMEOUT_MS
```

Comment:
Was wondering what we were passing this into, and TIL we are using the old `request` lib here... which was deprecated in 2020 lol. And Bluebird in the other file, man.